### PR TITLE
Create Python entry for random.random()

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Create an entry for python/concepts/random-module/terms/random

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by SSwiniarski** [comment]
This is an orphan from the original assignment sheet.

File: content/python/concepts/random-module/terms/random/random.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.random()` method generates a pseudo-random floating-point number between 0 and 1. It takes no parameters.
+
+It can be useful to create a pseudo-random float less than a given number just by multiplying that number by `random.random()`.
+
```
Comment (KTom101) [inline content/python/concepts/random-module/terms/random/random.md:19]: Just to follow the general outline of the other "Random Module" term entries. Add a  "Syntax" section before the "Example"  section.

File: content/python/concepts/random-module/terms/random/random.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+Title: 'random()'
```
Comment (KTom101) [inline content/python/concepts/random-module/terms/random/random.md:2]:
```suggestion
Title: '.random()'
```

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski I've completed the first review on this entry. Thank you!

**Comment by SSwiniarski** [comment]
@KTom101, I just applied your suggestions.

**Review by KTom101 — APPROVED** [review]
Handing this to @yangc95 or @Dusch4593  for a second review!

File: content/python/concepts/random-module/terms/random/random.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.random()` method returns a pseudo-random floating-point number between 0 and 1. It takes no parameters.
+
+It can be useful to create a pseudo-random float less than a given number just by multiplying that number by `random.random()`.
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/random/random.md:18]:
```suggestion
```

I suggest moving this content down to the "Codebyte Example" section.

File: content/python/concepts/random-module/terms/random/random.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.random()` method returns a pseudo-random floating-point number between 0 and 1. It takes no parameters.
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/random/random.md:16]:
```suggestion
The `random.random()` method accepts no parameters and returns a pseudo-random floating-point number between 0.0 and 1.0.
```

File: content/python/concepts/random-module/terms/random/random.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+It can be useful to create a pseudo-random float less than a given number just by multiplying that number by `random.random()`.
+
+## Syntax
+
+```py
+random.random()
+```
+
+## Codebyte Example
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/random/random.md:26]:
```suggestion
## Codebyte Example

It is possible to generate a pseudo-random float between 0 and a given number by multiplying it by `random.random()`:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left you some comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions.

File: content/python/concepts/random-module/terms/random/random.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.random()` method accepts no parameters and returns a pseudo-random floating-point number between 0.0 and 1.0.
+
+## Syntax
+
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/random/random.md:19]:
```suggestion

After importing the `random` module, the `random.random()` method can be accessed:

```

Comment (Dusch4593) [reply content/python/concepts/random-module/terms/random/random.md:19]: This is totally not necessary but was wondering everyone else thought about at least mentioning this in the context of applying the syntax?

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski All LGTM! I left one more comment that is more of a question as opposed to a requested change. Feel free to resolve, commit or no commit. 😄

File: content/python/concepts/random-module/terms/random/random.md (hunk 6 above)
Comment (SSwiniarski) [reply content/python/concepts/random-module/terms/random/random.md:19]: @Dusch4593, I don't know. Isn't it implied by being an entry under "random-module?"

**Comment by SSwiniarski** [comment]
@Dusch4593, in regards to your question, I don't think it's necessary.

File: content/python/concepts/random-module/terms/random/random.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+```py
+random.random()
+```
+
+## Codebyte Example
+
+It is possible to generate a pseudo-random float between 0 and a given number by multiplying it by `random.random()`:
+
+```codebyte/py
```
Comment (Dusch4593) [inline content/python/concepts/random-module/terms/random/random.md:28]:
```suggestion
```codebyte/python
```

**Comment by Dusch4593** [comment]
@SSwiniarski Had one last suggestion for this entry. 

@yangc95 @KTom101 Was there anything else to check before merging?

**Comment by SSwiniarski** [comment]
@Dusch4593 done with last suggestion

**Comment by Dusch4593** [comment]
@SSwiniarski The entry is now live: https://www.codecademy.com/resources/docs/python/random-module/random

**Pull request merged**